src/pat/fullscreen/fullscreen-close.js

```diff
@@ -0,0 +1,20 @@
+define([
+    "pat-base",
+    "screenful"
+], function(Base, screenful) {
+    return Base.extend({
+        name: "fullscreen-close",
+        trigger: ".close-fullscreen",
```

Comment:
Attention: this is against the convention of using the pattern name ``pat-fullscreen-close`` as trigger. But since this makes only sense in the context of this fullscreen pattern this might be OK.